doc/dataset/modules/ROOT/pages/using-provider.adoc

```diff
@@ -45,6 +48,20 @@ Example of creating 10 new realms:
 ./dataset_import.sh -r 10
 ----
 
+Alternatively, the user could also run the link:{github-files}/provision/minikube/Taskfile.yaml[Taskfile] from within the provision/minikube module to execute the `dataset-import` task, to seamlessly execute the automation for Kubernetes style deployments.
+
+The `dataset-import` task re-uses the command line arguments defined above to make it convenient for the user.
+Note the two dashes (`--`) that separate the task name from the command line options passed to the script.
+
+[source,bash]
+----
+task dataset-import -- -r 10
+----
+
```

Comment:
In my review commit I found that an example plus a link to the docs would help the user to understand what to do here.

Reply:
Thank you for adding the example, it definitely helps with the user experience